system-tests/test/component_testing_spec.ts

```diff
@@ -106,3 +106,21 @@ describe(`React major versions with Webpack`, function () {
     })
   }
 })
+
+describe('experimentalSingleTabRunMode', function () {
+  systemTests.setup()
+
+  systemTests.it('executes all specs in a single tab', {
```

Comment:
I'm assuming no, but with our current system tests, this there any way to benchmark the performance gains from this flag?

Reply:
They are fairly simple at this point - I don't think the will reveal and isolation issues, but having some that do sure would be nice. I'm not entirely sure of the best example of one, though - something that modifies `window.top`? :thinking:

Reply:
As for performance, and tracking regressions, no... I think we should look into this, probably as a fast follow. One idea I had was we could just run `app-component-tests` twice using the system test infrastructure, once with the experimental flag and once without, then print a report and make some assertions. It might cost us a bit of extra $, will need to look into this more closely.

I think there's some talk of using Graphana in #team-e2e-testing but this seems like it won't land anytime soon.

I'll try to add something small in here to at least verify it is faster with the experimental flag.